dm/dm/proto/dmworker.proto

```diff
@@ -478,3 +479,16 @@ enum ValidationErrOp {
   ResolveErrOp = 2;
   ClearErrOp = 3;
 }
+
+message ValidationStats {
+    uint64 processedCount = 1;
+    uint64 succeededCount = 2;
+    uint64 errorCount = 3;
+}
+
+message BriefValidationStatus {
+    string mode = 1;
+    string lastCheckTime = 2;
+    string status = 3;
```

Comment:
seems this is the `Stage` type.

And I don't know what's the difference of "status", "stage" :joy:

Reply:
They are definitely the same thing. I just followed the naming in the spec...Should I replace it with "Stage" (I bet @sunzhaoyang won't complain of it)?

Reply:
As my option.

Stage means different phases of a process, each phase finish different jobs, e.g. ‘dump/load/sync’ 

Status means different situation of the same object, e.g. ‘Running/Pause/Stop’

But  I think we had mixed them in the previous  features, so following the tradition is ok too. The reviewer can decide it.